admin/controllers/import.php

```diff
@@ -88,19 +92,20 @@ private function CsvImport($type, $dbname)
 			// search for bom - then it is utf-8
 			$bom = pack('CCC', 0xEF, 0xBB, 0xBF);
 			$fc = fread($handle, 3);
-			$convert = strncmp($fc, $bom, 3) !== 0;
-			if ($convert) {
+			$convert = strncmp($fc, $bom, 3) == 0;
```

Comment:
The meaning of $convert has been reversed from "no bom found" to opposite "bom found"